lib/rucio/core/authentication.py

```diff
@@ -385,23 +411,43 @@ def delete_expired_tokens(total_workers, worker_number, limit=1000, *, session:
     # get expired tokens
     try:
         # delete all expired tokens except tokens which have refresh token that is still valid
-        query = session.query(models.Token.token).filter(and_(models.Token.expired_at <= datetime.datetime.utcnow()))\
-                                                 .filter(or_(models.Token.refresh_expired_at.__eq__(None),
-                                                             models.Token.refresh_expired_at <= datetime.datetime.utcnow()))\
-                                                 .order_by(models.Token.expired_at)
+        query = select(
```

Comment:
I would have hoped that there is an easier way to perform a simple deletion :thinking: but it's OK for me to keep it this way